uni-bern-institut-fur-musikwissenschaft-note.csl

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" names-delimiter=" / " demote-non-dropping-particle="never" default-locale="de">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Uni Bern Institut für Musikwissenschaft (note)</title>
+    <id>http://www.zotero.org/styles/uni-bern-institut-fur-musikwissenschaft-note</id>
+    <link rel="self" href="http://www.zotero.org/styles/uni-bern-institut-fur-musikwissenschaft-note"/>
+    <!--Stilvorgaben des Instituts für Musikwissenschaft -->
+	<link rel="documentation" href="https://www.musik.unibe.ch/unibe/portal/fak_historisch/dkk/musik/content/e39903/e561201/e561204/Wissenschaftliches_Arbeiten_Bern_20171208_ger.pdf"/>
+	<!-- Erläuterung zur Umsetzung in CSL-->
+	<link rel="documentation" href="https://www.musik.unibe.ch/unibe/portal/fak_historisch/dkk/musik/content/e39903/e561201/e884863/Zotero-ZitierstilIMWUBBern-ReadMe20190612_ger.pdf"/>
+    <contributor>
+      <name>Denis Maier</name>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <updated>2019-06-03T11:32:26+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="verb-short">hrsg. v.</term>
+      <term name="anonymous" form="short">o.A.</term>
+      <term name="number-of-volumes" form="short">Bde.</term>
+      <term name="sub verbo" form="short">Art.</term>
+      <term name="issue" form="short">H.</term>
+      <term name="and">u.</term>
+    </terms>
+    <date form="numeric">
+      <date-part name="day" suffix="."/>
+      <date-part name="month" form="numeric" suffix="."/>
+      <date-part name="year"/>
+    </date>
+  </locale>
+  <macro name="sort">
+    <choose>
+      <if match="none" variable="author composer editor">
+        <text macro="anon"/>
```

Comment:
If you want anonymous works first, I'd make this explicity (i.e. something like `text value="1" and then `text value="2"` else -- both for legibility and because I'm not 100% sure the specs are clear on which way this should sort.

Reply:
Ok. Changed that

Reply:
Oh. After some discussions the department asked for "o.A." in the author field for anonymous works... So, these macros shouldn't be need anyway.